### ISSUE TITLE
Missing functions and readme is cluttered

### ISSUE THREAD

**Issue by belldandu** [TheRealAksel/structjs#1]
I see unpack_from, pack_into and size but i dont see pack, unpack and iter_unpack. 
https://docs.python.org/3/library/struct.html

I also find it slightly hard to read your readme.

**Comment by lyngklip** [comment]
The omission of pack and unpack was intentional. I use pack_into and unpack_from exclusively. pack and unpack can be achieved in terms of pack_into and unpack_from. I never used Python 3, hence no iter_unpack. I see iter_unpack applies unpack_from repeatedly. It shouldn't be hard to implement. I may do that. There's no calcsize either.

I'm sorry you are having trouble reading the readme. As far as I recall I copied most of it blatantly off the Python docs.

Feel free to suggest improvements.

edit:
I have added functions pack, unpack, pack_into, unpack_from, calcsize, and iter_unpack to structjs.
I have added methods pack, unpack, and iter_unpack to objects returned by the struct function.
The test suite is not as thorough as I should like.
